dev/merge_spark_pr.py

```diff
@@ -285,6 +286,56 @@ def resolve_jira_issues(title, merge_branches, comment):
     for jira_id in jira_ids:
         resolve_jira_issue(merge_branches, comment, jira_id)
 
+def standardize_jira_ref(text):
+    # Standardize the [MODULE] SPARK-XXXXX prefix
```

Comment:
It would be pretty cool to add a doctest here.  I've generally found doctests to be pretty helpful when developing parsing / string manipulation stuff like this, such as the pull request title parser that I wrote for spark-prs.appspot.com: https://github.com/databricks/spark-pr-dashboard/blob/0ee9102b7bb30fa61e2f47c3d282c5a018a7d198/sparkprs/utils.py#L55

Unfortunately, this script isn't well-structured to support doctests because the main code isn't in a method.  If you want to test this, we can move the lines below this function (starting on 291/341) into a new main() function, then call that from a `if __name__ == '__main__' block.  It will blow up the size of the diff a bit, but that's not a huge deal since we don't backport changes to this file anyways.